src/java/za/co/mip/ablduck/javadoc/JavadocListener.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package za.co.mip.ablduck;

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "JavadocListener.java" should be located in "za/co/mip/ablduck" directory, not in "/ebs/ebs1/jenkins/workspace/PCTNewJob_PR-247-X33Q424TCA4ZMZR72S4Y6PBXSKT7FL6QYGBFILF26W7CMTBRNBCA/src/java/za/co/mip/ablduck/javadoc". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1598)